packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -369,6 +376,12 @@ For more information see: ${supportedFlutterVersionsUrl.toLink()}''');
   /// specified by the user is not found/supported.
   Future<String> resolveTargetFlutterRevision() async {
     if (flutterVersionArg == 'latest') return shorebirdEnv.flutterRevision;
+    if (flutterVersionArg == 'system') {
+      return flutterVersionFromSystemFlutter(process);
```

Comment:
```suggestion
      return shorebirdFlutter.getSystemVersion()
```

Reply:
That returns Future<String?> instead, so keeping the new version for now.